src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/FeatureConditionEvaluation.java

```diff
@@ -25,8 +27,12 @@ public boolean preHandle(HttpServletRequest request,
 
             throw new AccessDeniedException(ERROR_MESSAGE_403_FORBIDDEN);
         }
-        return true;
-    }
 
+        if (Boolean.FALSE == activeOrgsExternalEnabled && request.getRequestURI()
```

Comment:
'Boolean.FALSE ==' condition is not required since activeOrgsExternalEnabled is boolean (primitive)

Reply:
Added here: https://github.com/hmcts/rd-professional-api/pull/694/commits/202f8448fbe660b8059e9da280ecf046159dd383